elastic/integrations
# Update ECS version from 8.3.x & 8.4.x to 8.5.1

- Enhancement

## What does this PR do?

Updates ECS version to 8.5.1

## Checklist



## How to test this PR locally

Verify the ECS version is properly updated in all the places. Execute the system and pipeline tests.

## Related issues

- Closes #4791
